app/views/bookings/show.html.erb

```diff
@@ -0,0 +1,95 @@
+<% provide :title, t(".title") %>
+<h1><%= t ".title" %></h1>
+<div class="row">
+  <div class="col-md-5 col-md-offset-4">
+    <table class="booking-info">
+      <tr>
+        <th><%= t ".customer_name" %></th>
+        <td><%= @user.name %></td>
+      </tr>
+      <tr>
+        <th><%= t ".tour_name" %></th>
+        <td><%= @tour.name %></td>
+      </tr>
+      <tr>
+        <th><%= t ".number_passengers" %></th>
+        <td><%= @booking.number_passengers %></td>
+      </tr>
+      <tr>
+        <th><%= t ".locations" %></th>
+        <td>
+          <% @tour.locations.each do |location| %>
```

Comment:
`@tour.locations` có bị N+1 query ko thanh niên ơi

Reply:
@VuThiTranVan Dạ đoạn đó em chỉ dùng để in ra tên location thôi ạ nên hông có bị N+1 chị ạ